src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -166,6 +167,13 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
         filteredPersons.setPredicate(predicate);
     }
 
+    @Override
+    public void sortFilteredPersonList(Comparator<Person> comparator) {
+        requireNonNull(comparator);
+        ObservableList<Person> sortedList = this.addressBook.getPersonList().sorted(comparator);
+        filteredPersons = new FilteredList<>(sortedList);
+    }
+
```

Comment:
The sort will cause the filtered list to reset and will return a full sorted list, is it possible to keep sort the already filtered list?